### PR TITLE
firefox-esr: update to 52.7.3.

### PR DESCRIPTION
**Pull request by cr6git**

### CODE REVIEW

**Comment by maxice8** [comment]
don't forget about `xtrimdeps` ;)

**Comment by Gottox** [comment]
:+1: Let's see how ff-59 builds and update esr afterwards.

**Comment by cr6git** [comment]
@maxice8 I trimmed even more.

**Comment by cr6git** [comment]
@Gottox Upated to the latest release.

File: srcpkgs/firefox-esr/template [hunk 1]
```diff
@@ -25,19 +24,12 @@ makedepends="
 depends="nss>=3.27 desktop-file-utils hicolor-icon-theme"
 conflicts="firefox>=0"
 
-build_options="alsa dbus gtk3 rust pulseaudio startup_notification xscreensaver"
```
Comment (Gottox) [inline srcpkgs/firefox-esr/template]: Why remove rust?

Comment (cr6git) [reply srcpkgs/firefox-esr/template]: Useless.

Comment (Gottox) [reply srcpkgs/firefox-esr/template]: why useless?

Comment (Gottox) [reply srcpkgs/firefox-esr/template]: I'm not a fan of disabling rust if there's no need to. Rust is basicly a security feature of firefox. See https://hacks.mozilla.org/2016/07/shipping-rust-in-firefox/

Comment (cr6git) [reply srcpkgs/firefox-esr/template]: I currently run this version compiled locally on x86_64-musl, it works flawlessly.
For reference: in [FreeBSD](https://www.freshports.org/www/firefox-esr) `RUST=of`

Comment (cr6git) [reply srcpkgs/firefox-esr/template]: @Gottox for security matters? oh! it'a okay, I will amend my commit. :innocent:

Comment (cr6git) [reply srcpkgs/firefox-esr/template]: @Gottox I asked the same question to **asergi** [here](https://github.com/voidlinux/void-packages/pull/6760#issuecomment-388053697)... Why rust is enabled only for x86_64*? Not available for 32-bit? Thanks.

Comment (cr6git) [reply srcpkgs/firefox-esr/template]: I found the solution.

**Pull request merged**